### ISSUE TITLE
Custom Trades Don't Work on Sponge Server Framwork

### ISSUE THREAD

**Issue by lachian** [crazysnailboy/VillagerTrades#19]
See https://github.com/SpongePowered/SpongeForge/issues/2067. 

Is there anyway this mod can accomplish what it wants to without using reflection?

**Comment by crazysnailboy** [comment]
I don't think so, unfortunately. I need to modify the trades collection on the VillagerCareer class, which is private and that class' getTrades method returns an unmodifiable version of the private list. The only other way I could do it I think would be to use ASM or an access transformer, both of which are probably hackier solutions than using reflection.

I notice that there's a new commit referencing the above issue and that it's now been closed. Does that change fix this issue?

**Comment by CrypticDivide** [comment]
No, the issue has not been fixed entirely, but the problem is beyond your control.

The villager trades get loaded into the villagers, but because of how Sponge is importing trades from both its own system and Forge's system, the vanilla trades are still there.
